core/base-service/errors.js

```diff
@@ -42,6 +42,7 @@ class ShieldsRuntimeError extends Error {
     if (props.underlyingError) {
       this.stack = props.underlyingError.stack
     }
+    this.cacheSeconds = props.cacheSeconds
```

Comment:
I've done this here so any `ShieldsRuntimeError` can have a custom `cacheSeconds` but at the moment we're only using it for `Inaccessible`